kani-driver/src/args/mod.rs

```diff
@@ -79,6 +79,9 @@ pub struct StandaloneArgs {
 
     #[command(subcommand)]
     pub command: Option<StandaloneSubcommand>,
+
+    #[arg(long, hide = true, requires("enable_unstable"))]
```

Comment:
`enable_unstable` is deprecated. Also, do we really need an unstable attribute here?

Reply:
Removed this requirement.